app/models.py

```diff
@@ -40,11 +40,16 @@ class DatasetModel(db.DynamicDocument):
     def save(self, *args, **kwargs):
 
         directory = os.path.join(Config.DATASET_DIRECTORY, self.name + '/')
+        thumbnails = os.path.join(Config.DATASET_THUMBNAILS, self.name + '/')
```

Comment:
Is there a reason for having a special directory for the thumbnails? Why not mount them inside a dataset directory? Like: `/datasets/<dataset_name>/_thumbnails/<thumbnail_file>` ?

Reply:
The thumbnails won't be automatically load to the interface if the folder is into the dataset folder? That's why you put the underscore?

Reply:
[Nope](https://github.com/jsbroks/coco-annotator/blob/f2368170799f2f20ddcd51bff0cad2b60c2b1654/app/watcher.py#L26), I was thinking of switching it to a dot `.thumbnails` which is for hidden folders.

Reply:
Excellent, you can switch it and then we just have to change the path in the config file and docker volumes.